Gemfile

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-ruby '>= 3.2.0', '< 3.5'
+ruby '>= 3.2.0'
```

Comment:
I would probably keep this, as we otherwise may run into a similar issue next year when 3.5 is released.